mne/report.py

```diff
@@ -1582,6 +1596,19 @@ def _render_raw(self, raw_fname):
             div_klass='raw', id=global_id, caption=caption, info=raw.info,
             meas_date=meas_date, n_eeg=n_eeg, n_grad=n_grad, n_mag=n_mag,
             eog=eog, ecg=ecg, tmin=tmin, tmax=tmax)
+
+        if isinstance(self._raw_psd, dict):
+            from matplotlib.backends.backend_agg import FigureCanvasAgg
+            n_ax = sum(kind in raw for kind in _data_types)
+            fig, axes = plt.subplots(n_ax, 1, figsize=(6, 1 + 1.5 * n_ax),
+                                     dpi=92)
+            FigureCanvasAgg(fig)
```

Comment:
why not `_figure_agg` here?

Reply:
Because it won't work with `plt.subplots` call